vip-portal/src/main/resources/vm/gwendia-standalone.vm

```diff
@@ -5,78 +5,99 @@
     <source name="results-directory" type="URI" optional="false" default="/vip/Home" pretty-name="Results directory">
         <source-comment>Directory where the results will be stored.</source-comment>
     </source>
-#foreach( $input in $tool.getInputs() )
-  #set($type="string")
-  #if($input.getType().getCamelName()=="File")
-    #set($type="URI")
-  #end
-  #if($input.getType().getCamelName()=="Flag")
-    #if($input.getDefaultValue() && $input.getDefaultValue()!="")
-      <source name="$input.getId()" type="string" optional="$input.isOptional()" default="$input.getDefaultValue()" vip-type-restriction="flag" pretty-name="$input.getName()">
-    #else
-      <source name="$input.getId()" type="string" optional="$input.isOptional()" default="false" vip-type-restriction="flag" pretty-name="$input.getName()">
+    #foreach( $input in $tool.getInputs() )
```

Comment:
I thnk VS or something else changed the indentation of velocity code and it generetad a lot of diff.
Could it be possible to keep the original indentation ?

Reply:
indeed, restore original indentation.